### PR TITLE
readme.md: Minor formatting

### PR DESCRIPTION
**Pull request by Stevoisiak**
* Formatted filenames with code blocks
* ~~Added file extensions for script files~~
* ~~Use forward slash for directory paths~~

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -88,12 +88,12 @@ All libs listed here (bar cmake) required in 32 bit variants.
...
 1. Check out the repository. This can be done using [GitHub Windows](https://windows.github.com/) or [other tools](https://help.github.com/articles/which-remote-url-should-i-use).
 2. Open a new PowerShell window and navigate to the repository.
-3. Run the setenv.ps1 script in the repository to setup your PowerShell environment for OpenRCT2 development. This will warn you of any missing applications required to build OpenRCT2.
-4. Run ```install``` to download the required dependencies to build OpenRCT2.
-5. Run ```build all``` to build all the required components for OpenRCT2.
-6. Run ```run``` to run OpenRCT2.
+3. Run the ```setenv.ps1``` script in the repository to setup your PowerShell environment for OpenRCT2 development. This will warn you of any missing applications required to build OpenRCT2.
+4. Run ```./scripts/ps/install.ps1``` to download the required dependencies to build OpenRCT2.
+5. Run ```./scripts/ps/build.ps1``` to build all the required components for OpenRCT2.
+6. Run ```./scripts/ps/run.ps1``` to run OpenRCT2.
```
Comment (IntelOrca) [inline readme.md]: Why have you changed these?

File: readme.md [hunk 2]
```diff
@@ -88,12 +88,12 @@ All libs listed here (bar cmake) required in 32 bit variants.
...
 
-These PowerShell scripts are stored in ```.\scripts\ps``` and have parameters. Once you have use the build script once, further development can be done within Visual Studio by opening ```openrct2.sln```. The build scripts have several commands allowing you to rebuild certain components such g2.dat or language files.
+These PowerShell scripts are stored in ```./scripts/ps``` and have parameters. Once you have use the build script once, further development can be done within Visual Studio by opening ```openrct2.sln```. The build scripts have several commands allowing you to rebuild certain components such ```g2.dat``` or language files.
```
Comment (IntelOrca) [inline readme.md]: Given these are instructions for Windows, why are you changing the slash direction?

Comment (Stevoisiak) [inline readme.md]: I figured I'd do it for consistency's sake. I can change that back easilly

Comment (IntelOrca) [inline readme.md]: ok

File: readme.md (hunk 1 above)
Comment (Stevoisiak) [inline readme.md]: I thought it might be slightly more intuitive to run the script files directly. I'll change that back.

**Pull request merged**